ReactWindows/ReactNative/Modules/Clipboard/ClipboardModule.cs

```diff
@@ -0,0 +1,88 @@
+﻿using ReactNative.Bridge;
+using System;
+using DT = Windows.ApplicationModel.DataTransfer;

```

Comment:
Not to be too pedantic, but can we just expand the acronym to DataTransfer, I know it's confusing for some to read code with acronyms.